interfaces/automated-testing/geo-awareness/geo-awareness.yaml

```diff
@@ -61,5 +124,83 @@ paths:
           description: The access token was decoded successfully but did not include a scope appropriate to this endpoint.
         '404':
           description: The USS automated testing interface is not activated.
-      summary: Retrieve the status of the USS automated testing interface
-      description: Get the status of the USS automated testing interface
+      summary: Status of the USS automated testing interface
+      description: Get the status of the USS automated testing interface.
+
+
+  /v1/geozones/{geozones_id}:
+    parameters:
+      - name: geozones_id
+        in: path
+        required: true
+        description: A UUID string identifying the injected geozones dataset.
+        schema:
+          $ref: '#/components/schemas/UUIDv4Format'
+
+    put:
+      security:
+        - Authority:
+            - utm.inject_test_data
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InjectGeozonesRequest'
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InjectGeozonesResponse'
+          description: >-
+            Request has been correctly handled and the geozones dataset will be imported and activated by the USS.
+        '401':
+          description: Bearer access token was not provided in Authorization header, token could not be decoded, or token was invalid.
+        '403':
+          description: The access token was decoded successfully but did not include a scope appropriate to this endpoint.
+      operationId: PutInjectGeozones
+      summary: Activate a Geozones dataset
+      description: Instructs the USS to import and activate a Geozones dataset.
+
+    get:
+      security:
+        - Authority:
+            - utm.inject_test_data
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InjectGeozonesResponse'
+          description: >-
+            The Geo-Awareness dataset is handled by the USS and its status was retrieved successfully.
+        '401':
+          description: Bearer access token was not provided in Authorization header, token could not be decoded, or token was invalid.
+        '403':
+          description: The access token was decoded successfully but did not include a scope appropriate to this endpoint.
+        '404':
+          description: The Geozones dataset has been successfully deleted or didn't exist.
+      operationId: GetInjectGeozones
+      summary: Status of the Geozones dataset
+      description: Get the status of the Geozones dataset.
+
+    delete:
+      security:
+        - Authority:
+            - utm.inject_test_data
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InjectGeozonesResponse'
```

Comment:
Since we aren't actually performing an Inject operation here, this data type name doesn't seem appropriate (though I agree we should reuse the same data structure).  How about changing InjectGeozonesResponse -> GeozoneResponse?